packages/react/src/components/NodeWrapper/index.tsx

```diff
@@ -143,9 +144,12 @@ export function NodeWrapper<NodeType extends Node>({
       event.preventDefault();
 
       store.setState({
-        ariaLiveMessage: `Moved selected node ${event.key
-          .replace('Arrow', '')
-          .toLowerCase()}. New position, x: ${~~internals.positionAbsolute.x}, y: ${~~internals.positionAbsolute.y}`,
+        ariaLiveMessage:
+          labelConfig['a11yDescription.ariaLiveMessage']?.(
+            event.key.replace('Arrow', '').toLowerCase(),
+            ~~internals.positionAbsolute.x,
+            ~~internals.positionAbsolute.y
+          ) || '',
```

Comment:
to avoid this, the store labelConfig should have a slightly different type (all fields required)